packages/volto-razzle/package.json

```diff
@@ -38,32 +37,31 @@
     "file-loader": "^4.3.0",
     "fs-extra": "^9.1.0",
     "inquirer": "^7.3.3",
-    "jest": "^26.4.2",
     "mri": "^1.1.4",
     "null-loader": "^4.0.1",
     "pnp-webpack-plugin": "^1.6.4",
     "postcss": "^8.2.4",
-    "postcss-load-config": "^3.0.0",
-    "postcss-loader": "^4.2.0",
+    "postcss-load-config": "3.1.4",
+    "postcss-loader": "7.0.2",
     "process": "^0.11.10",
     "razzle-start-server-webpack-plugin": "4.2.18",
     "react-dev-utils": "^11.0.4",
-    "react-refresh": "^0.9.0",
+    "react-refresh": "0.14.0",
     "release-it": "^19.0.4",
     "resolve": "^1.17.0",
     "sade": "^1.4.2",
     "source-map-support": "^0.5.16",
     "string-hash": "^1.1.3",
-    "style-loader": "^2.0.0",
+    "style-loader": "3.3.1",
     "terminate": "^2.1.2",
-    "terser-webpack-plugin": "^2.3.5",
+    "terser-webpack-plugin": "5.3.6",
     "tiny-async-pool": "^1.1.0",
     "url-loader": "^2.3.0",
     "webpack-manifest-plugin": "^3.1.1",
     "webpackbar": "~4||~5"
   },
   "peerDependencies": {
-    "babel-preset-razzle": "4.2.18",
+    "@volto/babel-preset-razzle": "workspace:*",
```

Comment:
`@plone/scripts` also uses it:

https://github.com/plone/volto/blob/5bad102adaef46348a35d3c268605e28e6fabe23/packages/scripts/package.json#L45